.azure-pipelines/scripts/publish-codecov.sh

```diff
@@ -7,7 +7,8 @@ set -o pipefail -eu
 
 output_path="$1"
 
-curl --silent --show-error https://codecov.io/bash > codecov.sh
+curl --silent --show-error https://ansible-ci-files.s3.us-east-1.amazonaws.com/ansible-test/codecov.sh > codecov.sh
```

Comment:
Since this file is used by AZP, and not ansible-test, it should probably be in a different directory. Maybe this?

```suggestion
curl --silent --show-error https://ansible-ci-files.s3.us-east-1.amazonaws.com/azp/codecov.sh > codecov.sh
```

Reply:
I'll put it in a `codecov` directory as we discussed.